packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Switch.module.scss';
+import { enableFocusStyles, disableFocusStyles } from './_SwitchUtils';
+
+const cx = classNames.bind(styles);
+
+// Translations Required
+const SWITCH_STATE = Object.freeze({
+  ON: 'On',
+  OFF: 'Off',
+});
+
+const propTypes = {
+  /**
+   * Id of the Switch button.
+   */
+  buttonId: PropTypes.string.isRequired,
+  /**
+   * Id of the Switch label.
+   */
+  labelId: PropTypes.string.isRequired,
+  /**
+   * Whether or not the Switch is checked ("ON").
+   */
+  checked: PropTypes.bool,
+  /**
+   * Whether or not the Switch is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * The label of the Switch component.
+   */
+  label: PropTypes.node.isRequired,
+  /**
+   * Callback function when switch value changes from ON / OFF.
+   * Parameters: 1. switch value 2. event.
+   */
+  onChange: PropTypes.func,
+};
+
+const defaultProps = {
+  checked: false,
+  disabled: false,
+  onChange: undefined,
+};
+
+const Switch = (props) => {
+  const {
+    buttonId,
+    labelId,
+    checked,
+    disabled,
+    onChange,
+    label,
+    ...customProps
+  } = props;
+
+  const sliderButton = useRef();
+
+  const handleOnClick = useCallback(
+    (event) => {
+    // See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Button#Clicking_and_focus
+    // Button on Firefox, Safari and IE running on OS X does not receive focus when clicked.
+    // This will set focus on the button when clicked.
+      sliderButton.current.focus();
+      if (onChange) {
+        onChange(!checked, event);
+      }
+    },
+    [checked, onChange],
+  );
+
+  const switchClassNames = cx([
+    'switch',
+  ]);
```

Comment:
```suggestion
  const switchClassNames = cx(['switch']);
```
Since this contains only a single class, it can be changed as above

Reply:
I would suggest moving this to be in-line on the component. It doesn't look like there's any logic being done by defining it here.

Reply:
changed : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864